src/translations/en.json

```diff
@@ -61,10 +61,10 @@
     "alarm_control_panel": {
       "armed": "Armed",
       "disarmed": "Disarmed",
-      "armed_home": "Armed home",
-      "armed_away": "Armed away",
-      "armed_night": "Armed night",
-      "armed_custom_bypass": "Armed custom bypass",
+      "armed_home": "Armed Home",
```

Comment:
old style is correct: https://material.io/guidelines/style/writing.html#writing-capitalization-punctuation

Reply:
Ugh, it looks weird to me but I understand. Gotta follow the rules I suppose, I'll update with another commit.